schemas/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+)
+
+func main() {
+	if err := realMain(); err != nil {
+		log.Fatal(err)
+	}
+	fmt.Println("execution finished successfully")
+}
+
+func realMain() error {
+	fmt.Printf("downloading latest schema version...\n")
+	out, err := os.Create("schemas/downloaded.json")
```

Comment:
Suggests for the file name are appreciated. Perhaps it should be timestamped instead of constant?

Reply:
Time, or date stamp will make more sense down the road - right now this is good.